packages/aws-cdk-lib/aws-codebuild/README.md

```diff
@@ -490,6 +490,27 @@ new codebuild.Project(this, 'Project', {
 })
 ```
 
+### Attribute based compute
```

Comment:
The hyphen is also missing from the PR title, I think other instances are correct
```suggestion
### Attribute-based compute
```